src/Illuminate/Database/Migrations/MigrationCreator.php

```diff
@@ -67,7 +65,7 @@ public function create($name, $path, $table = null, $create = false)
     }
 
     /**
-     * Ensure that a migration with the given name doesn't already exist.
+     * Ensure that a migration with the given name does not already exist.
```

Comment:
Why change this line?

Reply:
Because typically contractions are only used in conversational English.

Reply:
I've never met a technical writer that dislikes contractions being used.

Reply:
Even if that were the case, why change it in this PR? Make a separate PR for it

Reply:
He wrote and rewrote a bunch of comments in this PR. Are you seriously that offended by it?

Reply:
I'm not saying it's right or wrong. Nor that it needs to be changed or not. I was simply adding to the idea of "formal writing" that I have yet (in like 6 years of working with technical writers) met one that dislikes their use. Which means, it can't be that informal if few who write professionally in the tech industry seems to really care in either direction.

Reply:
I'm not offended by anything? I'm just saying it's a useless change and not really in the scope of this PR

Reply:
This is a trivial change. Honestly one that was either automated by my IDE or I did inadvertently when reviewing comments. Glad to revert it if additional changes are required for this PR.